3rdparty/python/requirements.txt

```diff
@@ -9,6 +9,7 @@ chevron==0.14.0
 fasteners==0.16.3
 freezegun==1.2.1
 ijson==3.2.3
+libcst==1.4.0
```

Comment:
do we want to guard this from leaking into pants runtime (beyond the migration code)?
https://github.com/pantsbuild/pants/blob/777c2822c3eb5ba1b94b6cd6589870fd25331907/3rdparty/python/BUILD#L31-L44

Reply:
I think libcst is an excellent utility for migrations, of which this will not be the last. Allowing broader use of it seems like a plus to me. So I don't think we should restrict this dependency with dependency visibility rules.

Reply:
I was feeling this could be useful as well, raised the question to get a conscious decision for it though, as there's been an on-going effort to reduce the number of 3rd party dependencies (albeit controversial at times) in #19282, so it would be good to have a clear record on the motivation for each dependency and it's intended use, I think.